mobify/pikabu
# Remove `@support`

Status: **Ready for Review**
Reviewers: @ericawright @stewartyu @mikenikles @fractaltheory @jansepar @marlowpayne

Due to an issue in some browsers (i.e. Android 4.4.2, Galaxy S4 default browser), using `@support` breaks the `:not()` CSS pseudo selector. See here:

https://bugs.chromium.org/p/chromium/issues/detail?id=257695

In particular, this was an issue in our project here: https://mobify.atlassian.net/browse/CF-105

Also: Removed the Chrome specific fix from before, as I cannot
reproduce that particular issue anymore.
## Changes
- Removed `@support` block, replaced with Deckard classes for Safari
- Removed the Chrome specific fix, as it no longer appears to be an issue (as per described in #69)
## Jira Tickets:
- [CF-105](https://mobify.atlassian.net/browse/CF-105)
## Todos:
- ~~Unlink CF project from having this branch as a dependency~~ nevermind, the project package.json points to the commit, not the branch
## How to Test
- Clone this repo and checkout this branch
- Run `bower install` and `npm install`
- Run `grunt build`
- Open `examples/index.html` in a **Galaxy S4, Android 4.4.2** (i.e. Luna) browser
- Open the Pikabu nav
- Notice that the some of the headings in the Pikabu nav are `red`

**Note**: I've intentionally added a test case to the example's stylesheet (not the Pikabu's `scss`) that demonstrates the issue. The `red` text should be inheriting colors from their parent, but because of the presence of `@support`, the `:not()` selector that sets `color: inherit` isn't being respected.
